docs/docs/cmd/spo/roledefinition/roledefinition-remove.md

```diff
@@ -0,0 +1,36 @@
+# spo roledefinition remove
+
+Should remove roledefinition from web
+
+## Usage
+
+```sh
+m365 spo roledefinition remove [options]
+```
+
+## Options
+
+`-u, --webUrl <webUrl>`
+: URL of the site from which role should be removed
+
+`-i, --id <id>`
+: Role definition id
+
+`--confirm`
+: Don't prompt for confirming removing the role definition
+
+--8<-- "docs/cmd/_global.md"
+
+## Examples
+
+Remove the role definition from site  _https://contoso.sharepoint.com/sites/project-x_ with id _1_
+
+```sh
+m365 spo roledefinition remove --webUrl https://contoso.sharepoint.com/sites/project-x --id 1
+```
+
+Remove the role definition from site  _https://contoso.sharepoint.com/sites/project-x_ with id _1_ and don't prompt for confirmation
+
+```sh
+m365 spo roledefinition remove --webUrl https://contoso.sharepoint.com/sites/project-x --id 1 --confirm
+```
```

Comment:
Let's end the file with an empty line so that our code snippet detection logic will work as intended